pom.xml

```diff
@@ -136,7 +136,7 @@
     <antlr.version>4.6</antlr.version>
     <jsonpath.version>2.7.0</jsonpath.version>
     <quartz.version>2.3.2</quartz.version>
-    <calcite.version>1.29.0</calcite.version>
+    <calcite.version>1.31.0</calcite.version>
```

Comment:
calcite has new javacc/fmpp codegen plugin mojo that we need to use separately. this will take some time. if there's any immediate requirement plz comment.